src/pages/patterns/loading-pattern/index.mdx

```diff
@@ -0,0 +1,113 @@
+
+Never represent toast notifications, overflow menus, dropdown items, modals, and loaders with skeleton states. Elements inside a modal may have a skeleton state, but the modal itself should not.
+
+## Progressive loading
+
+Progressive loading is when a page loads in batches. The simplest view of the page loads first, followed by progressively more detailed batches until the entire viewport has loaded.
+
+The primary batch should show a page's basic structure (the skeleton state versions of the container-based components), data-based text (the skeleton state version of text) and non-data text. Following batches can include images, content outside of the viewport, interactive (action-based) components, and data-based text.
+
+Not all items need a skeleton state and instead can be expressed as negative or white space until they load. For example, a 600 x 600px image can be shown as a 600 x 600px area of white space until the full image loads.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a dashboard using skeleton states to demonstrate progressive loading.](/images/prog-loading-1.png)
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![Example of a dashboard using skeleton states to demonstrate progressive loading.](/images/prog-loading-2.png)
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={8}>
+
+![Example of a dashboard using skeleton states to demonstrate progressive loading.](/images/prog-loading-3.png)
+
+<Caption>
+  Example of a dashboard using skeleton states to demonstrate progressive
+  loading.
+</Caption>
+
+</Column>
+</Row>
+
+## Loading components
+
+A loading component is a rotating shape indicating the application is processing.
+
+Use cases include:
+
+- When data processing takes more than a few seconds but no longer than one minute.
+- When data from a user input is being saved or submitted
+
+### Loading
+
+The [loading component](/components/loading/usage) should be used when the entire page is processing. This is often applied after data is submitted or saved by the user.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a large loader in an application](/images/larger-loader.png)
+
+<Caption>Example of a large loader in an application</Caption>
+
+</Column>
+</Row>
+
+### Inline loading
+
+Use the inline loading component when a single component is processing. For example, when a user restarts a stopped application, an inline loading component indicates the system is processing the restart request.
+
+<Row>
+<Column colLg={8}>
+
+![Example of an inline loader](/images/inline-loading-animated.gif)
+
+<Caption>Example of an inline loader</Caption>
+
+</Column>
+</Row>
+
+## 'Load more' button
+
+A 'Load more' button should be used to extend a list where there only a small fraction of options are displayed. It can also be used in cases where the list of options is populated via a database. Using 'Load more' allows the data to load in progressive batches.
```

Comment:
```suggestion
A **Load more** button should be used to extend a list where there only a small fraction of options are displayed. It can also be used in cases where the list of options is populated via a database. Using **Load more** allows the data to load in progressive batches.
```